src/globalValueSets/Payment_ACH_Code.globalValueSet

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<GlobalValueSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customValue>
+        <fullName>PPD</fullName>
+        <default>false</default>
+        <label>Prearranged Payment and Deposit Entry (PPD)</label>
+    </customValue>
+    <customValue>
+        <fullName>CCD</fullName>
+        <default>false</default>
+        <label>Cash Concentration or Disbursement (CCD)</label>
+    </customValue>
+    <customValue>
+        <fullName>WEB</fullName>
+        <default>false</default>
+        <label>Web-initiated Entry (WEB)</label>
+    </customValue>
+    <customValue>
+        <fullName>TEL</fullName>
+        <default>false</default>
+        <label>Telephone-initiated Entry (TEL)</label>
+    </customValue>
+    <customValue>
+        <fullName>BOC</fullName>
+        <default>false</default>
+        <label>Back Office Conversion (BOC)</label>
+    </customValue>
+    <customValue>
+        <fullName>ARC</fullName>
+        <default>false</default>
+        <label>Accounts Receivable Conversion (ARC)</label>
+    </customValue>
+    <description>The Standard Entry Class (SEC) code used for this ACH transaction.</description>
```

Comment:
Hi @sam-knox I want to double check with you that this Global Picklist should also have the same description as the new `Payment_ACH_Code__c` field description. Please let me know if that's not the case, thank you!

Reply:
Yep, that's just fine @mldyang.